test/acceptance_test.exs

```diff
@@ -0,0 +1,15 @@
+defmodule AcceptanceTest do
+  use ExUnit.Case
+
+  describe "acceptance" do
+    test "the game works when run from the command line" do
+      Mix.Shell.Process.cmd("mix escript.build")
+      Mix.Shell.Process.cmd("./elixir_tic_tac_toe_basic")
+
+      assert_received {_, _, ["Welcome to Tic Tac Toe!\n"]}
+      assert_received {_, _, [_empty_board]}
```

Comment:
**question**: Is it valuable to check this? 

My feeling is that checking the welcome message might be good enough for now.